src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: How to use a wallet
+description: A guide explaining how to send, receive tokens and connect to web3 projects.
+lang: en
+---
+
+# How to: Use a wallet
+
+Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our How to: "Register" an Ethereum account.
+
+
+## Open your wallet
+
+You should see a dashboard that will likely show your balance and contain buttons to Send and Receive tokens. 
+
+## Receive cryptocurrency
+
+Do you want to receive crypto into your wallet?
+
+Each wallet has its own receiving Ethereum address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum address will always start with “0x”.
```

Comment:
An account isn't a wallet. A wallet can have many 'receiving addresses' 

Words are hard.... 😁

Reply:
This is targeted at absolute newcomers who cannot do this on their own which leads me to think its not the right timing to introduce the differences at this point.  I do address this later in FAQs as secondary information. Do you think thats confusing? @minimalsm